src/vmware/azext_vmware/_params.py

```diff
@@ -39,7 +44,7 @@ def load_arguments(self, _):
         c.argument('nsxt_password', help='NSX-T Manager password.')
         c.argument('accept_eula', help='Accept the end-user license agreement without prompting.')
         c.argument('network_block', help='A subnet at least of size /22. Make sure the CIDR format is conformed to (A.B.C.D/X) where A,B,C,D are between 0 and 255, and X is between 0 and 22.')
-        c.argument('mi_system_assigned', help='Enable a system assigned identity.')
+        c.argument('mi_system_assigned', help=clean_help(ResourceIdentityType.__doc__), arg_type=get_enum_type([identityType.value for identityType in ResourceIdentityType]), deprecate_info=c.deprecate(redirect='--identity', hide=True))

```

Comment:
May I ask why do you want to deprecate the `--mi-system-assigned` parameter and redirect it to `--identity`? Please note we need to follow the [managed_identity_command_guideline](https://github.com/Azure/azure-cli/blob/dev/doc/managed_identity_command_guideline.md) to design parameters related to managed identities

Reply:
I did not realize that was the standard. Sorry @jonathanhe-msft for the busywork. Please undo some of these changes. It looks like the standard was being followed.